asciidoc/modules/con_mg_changes-in-infinispan-cluster-manager.adoc

```diff
@@ -0,0 +1,27 @@
+[id="changes-in-infinispan-cluster-manager_{context}"]
+= Changes in Infinispan cluster manager
+
+The following section describes the changes in the Infinispan cluster manager.
+
+== Updates in custom configurations
+
+The Infinispan cluster manager is based on Infinispan 11.
+
+In {VertX} {4}, the clustering SPI has been redesigned. The subscription data model has changed. As a result of this, {VertX} {3x} nodes and {VertX} {4} nodes cannot be added together in the same Infinispan cluster manager.
+
+The {VertX} applications are not impacted by this change as the EventBus and SharedData APIs remain the same.
+
+If you have the custom configurations from Vertx. 3.x, update the following parameters in the configuration file in {VertX} {4}:
```

Comment:
@vietj Can we call it configuration file or just configurations?

Reply:
@sangeetaraghu I submitted a proposal for this